js/demo/lazy-loading.js

```diff
@@ -26,14 +26,15 @@ var EffecktLazyLoading = {
 
   bindUIActions: function() {
 
-    var self = this;
+    var self = this,
+      windowObj = $(window);
```

Comment:
I don't think we have much of a naming convention for jQuery/non-jQuery variables here (it at least isn't in the style guide) but a name like `windowObj` makes me think it's just `window`. What does @h5bp/effeckt-css think?